.github/actions/backend-dependencies/action.yml

```diff
@@ -24,7 +24,7 @@ runs:
       shell: bash
 
     - name: Cache vendor directory
-      uses: actions/cache@69d9d449aced6a2ede0bc19182fadc3a0a42d2b0
+      uses: actions/cache@v3
```

Comment:
Isn't it better to fix the commit sha than the tag ?

Reply:
I think we'll may need to use tag instead : https://github.com/actions/checkout/releases/tag/v3.5.0

Reply:
I've restored v3 for consistency with develop branch